Tasks/app-store-promote/task.json

```diff
@@ -65,6 +65,34 @@
             "helpMarkDown": "The password of the account used to publish to the Apple App Store.",
             "visibleRule": "authType = UserAndPass"
         },
+                {
+            "name": "isTwoFactorAuth",
```

Comment:
This is only being used for auth type username/password and not endpoint. It seems like overall experience would be better to always show this checkbox and use it to determine if fastlane session field should be shown to the user or not.